src/markdown-pages/instant-observability/build-a-quickstart/create-a-quickstart.mdx

```diff
@@ -0,0 +1,452 @@
+
+```yml
+ 
+# Name of the pack (required)
+name: flashdb
+
+# Description of the pack (required)
+description: |
+  Flashdb is the fastest database in the space. 
+  The flashdb quickstart allows you to get visibility into the performance of flashdb with the help of following features:
+
+  Dashboards:
+  - Database methods: Monitor the number of transactions for each database method
+  - Database size: Monitor the size of database
+  - Average response time: Monitor the average response time for each type of database transaction
+  - Cache hits: Monitor the cache hits
+  - Errors: Monitor the errors occured for each type of database transaction
+  - Database keys: Monitor the keys for database
+ 
+  Alerts:
+  - Cache hit ratio: Alert is triggered when cache hit ratio falls below the set limit.
+  - Read response: Alert is triggered when read responses are slow and exceeds a set limit
+
+summary: |
+  Flashdb is the fastest database in the space. 
+  This quickstart allows you to get visibility into the performance of flashdb.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Design
+icon: icon.jpeg
+logo: logo.png
+
+# Authors of the pack (required)
+authors:
+  - {"enter your name"}
+
+# References to other packs (optional)
+# References are grouped by the type: operating-system, database, webserver, queue, ..
+# The reference under the type is a at least one system, so at least one operating-system, database, ..
+references:
+  - type: database
+    name:
+    - flashdb
+```
+
+Here's how your final _flashdb_ quickstart folder should look like.
+![quickstart description](../../../images/instant-observability/quickstart-description.png)
+
+## Contribute quickstart to GitHub
+
+Your quickstart is now ready to be published. You're going to commit your changes back to GitHub where it will be reviewed by New Relic. 
+Follow the [conventional commit syntax](https://github.com/newrelic/newrelic-observability-packs/blob/alerts-doc-update/CONTRIBUTING.md#using-conventional-commits) for New Relic to commit your changes.
+
+```bash
+ git add -A
+ git commit -m "feat(flashdb): Added a quickstart for flashdb"
+```
+
+<Callout variant="important">
+
+We, at New Relic, are always happy to receive contributions from the community. However, we are expecting a number of contributions for flashdb following the same lab course. 
+To help us manage your contributions efficiently, we encourage you to label your contribution following this particular lab as **lab** using `git tag` command.
+
+</Callout>
+
+Push your changes to GitHub:
+```bash
+ git Push
+```
+
+Once you've pushed your changes to GitHub, you can [create a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+
+![pull request](../../../images/instant-observability/pull-request.png)
+
+Submit and wait for the review. Thanks a lot for your submission!
```

Comment:
````
<Callout variant="important">

At this stage, your quickstart will be automatically validated in GitHub. If this were a real-life submission, it would then be approved and incorporated into New Relic I/O for your users to see. As this is a lab, we will not actually review this PR. Feel free to delete it after it validates your work.

</Callout>
````

Just to help them understand more about the validation in GitHub actions.